prefork/prefork.go

```diff
@@ -0,0 +1,179 @@
+package prefork
+
+import (
+	"flag"
+	"net"
+	"os"
+	"os/exec"
+	"runtime"
+
+	"github.com/valyala/fasthttp"
+	"github.com/valyala/fasthttp/reuseport"
+)
+
+const preforkChildFlag = "prefork-child"
+const defaultNetwork = "tcp4"
+
+var Child bool
```

Comment:
I did it  public for know if the current thread/worker is a child or not, because sometimes could be useful to change some things if it's a master or a worker.
You could check the `preforkChildFlag` but you need to know the name of it and parse the input, so what do you think about this solution:

```go
var child bool

func isChild() bool {
	return child
}
```

Reply:
What do you think about it?